src/components/FiscalYear/FiscalYearDetails/FiscalYearDetailsContainer.js

```diff
@@ -132,6 +137,15 @@ FiscalYearDetailsContainer.manifest = Object.freeze({
     fetch: false,
   },
   fyGroupSummaries: groupSummariesResource,
+  fyLedgers: {
+    ...ledgersResource,
+    accumulate: true,
+    fetch: false,
+    params: {
```

Comment:
Is it better to pass params in the place where you fetch data?

Reply:
if they are not variable, so why not to describe them in manifest?

Reply:
🆗, it works anyway)